code/api/views.py

```diff
@@ -366,7 +365,7 @@ def post(self, request, author_id):
                 else:
                     raise ValueError("Unknown object for like")
                 
-                if context_object.likes.filter(id=liking_author.id).exists():
```

Comment:
I definitely had this before and multi-likes were being handled correctly when I tested it. I must have messed up and reverted the changes when making that last-minute merge of the URL parsing code. Thanks for checking this.

Reply:
I agree tests would be good. Thanks